packages/cli/src/commands/certs/issue.ts

```diff
@@ -175,14 +175,12 @@ async function runStartOrder(
   );
   const [header, ...rows] = dnsTable(
     pendingChallenges.map(challenge => {
-      const parsedDomain = parse(challenge.domain);
-      if (parsedDomain.error) {
+      const subdomain = getSubdomain(challenge.domain);
+      if (!subdomain) {
```

Comment:
Hi, I noticed that the behavior after switching to `tldts` is different than with `psl`. There is now a migration guide which explains how to switch from `psl` to `tldts` while preserving the same behavior: https://github.com/remusao/tldts?tab=readme-ov-file#migrating-from-other-libraries

In particular, I think you might want to use the `{ allowPrivateDomains: true }` option to consider suffixes from the private section as well (as opposed to ICANN only).

I hope that helps!